examples/wmt/train.py

```diff
@@ -469,6 +469,12 @@ def decode_tokens(toks):
   target_shape = (config.per_device_batch_size, config.max_target_length)
 
   m = models.Transformer(eval_config)
+
+  # Calling `jit` on init will make JAX trace through the model and compile it
```

Comment:
I'm wondering about the discoverability of this explanation.
Wouldn't it make more sense to add a jit snippet with this explenation to the init method itself?

Reply:
Yes I liked that idea. I've moved it there now.